examples-staging/assets-local/keystone.ts

```diff
@@ -7,10 +7,26 @@ export default config({
     url: process.env.DATABASE_URL || 'file:./keystone-example.db',
   },
   lists,
-  images: {
-    upload: 'local',
-  },
-  files: {
-    upload: 'local',
+  storage: {
+    my_images: {
+      kind: 'local',
+      type: 'image',
+      generatedUrl: path => `http://localhost:3000/images${path}`,
```

Comment:
```suggestion
      generateUrl: path => `http://localhost:3000/images${path}`,
```


This name isn't amazing